docs/stratis.txt

```diff
@@ -20,8 +20,16 @@ and the filesystems allocated from the pool.
 Stratis creates a *pool* from one or more block devices (*blockdevs*), and
 then enables multiple *filesystems* to be created from the pool.
 
-If the block device is already in use, its signature has to be erased using
-*wipefs -a <blockdev>* before trying to add it to a pool.
+Stratis may refuse to use a block device for any of the following reasons:
+1. The block device could already be in use, in which case its signature may
```

Comment:
For point 1, I'd like to see something even more careful like `The block device could already be in use in another context outside of stratisd, in which case its signature may be erased using *wipefs -a <blockdev>*. To allow use with stratisd, the filesystem must be wiped prior to adding it to a pool, but this should only be done after careful consideration about whether or not the existing data on the device is critical and/or replicated elsewhere.`

Thoughts, @mulkieran?

Reply:
I'ld rather drop it from the description entirely. Its position  makes this text too prominent.

Maybe we can make a separate heading, just below ```ENVIRONMENT VARIABLES```. Call it something very generic like ```NOTES``` for now. Then say something like:

```
If a block device appears to be already in use, stratisd will refuse to claim it. To allow use with
stratisd, any signature on the device must first be erased. Please carefully verify the identity
and availability of the device before taking such a step.
```

I'ld rather leave multipath specifically out of the discussion entirely, basically because there are so many other tricky possibilities besides multipath to worry about.